packages/lesswrong/components/dialogues/ActiveDialogues.tsx

```diff
@@ -6,25 +6,44 @@ import { useCurrentUser } from '../common/withUser';
 import { postGetEditUrl } from '../../lib/collections/posts/helpers';
 import { Link } from '../../lib/reactRouterWrapper';
 import { useLocation } from '../../lib/routeUtil';
+import classNames from 'classnames';
+import VisibilityOff from '@material-ui/icons/VisibilityOff';
+import Visibility from '@material-ui/icons/Visibility';
+import { useUpdateCurrentUser } from '../hooks/useUpdateCurrentUser';
+
 
 const styles = (theme: ThemeType) => ({
   root: {
     marginLeft: "1%",
     display: "flex",
+    flexDirection: "row",
+    alignItems: "center",
     [theme.breakpoints.down('sm')]: {
       display: 'none'
     }
   },
   activeDot: {
     height: 8,
     width: 8,
-    backgroundColor: theme.palette.secondary.light,
     borderRadius: '50%',
     display: 'inline-block',
-    boxShadow: `0 0 5px ${theme.palette.secondary.light}, 0 0 8px ${theme.palette.secondary.light}, 0 0 11px ${theme.palette.secondary.light}`,
     marginRight: 11,
     marginLeft: 15
   },
+  collapsedDot: {
+    marginRight: 8,
+    marginLeft: 8
+  },
+  greenDot: {
+    backgroundColor: theme.palette.secondary.light,
+    boxShadow: `0 0 5px ${theme.palette.secondary.light}, 0 0 8px ${theme.palette.secondary.light}, 0 0 11px ${theme.palette.secondary.light}`,
+  },
+  orangeDot: {
+    backgroundColor: '#fdbd48',
+    border: `1px solid #fdbd48`,
+    boxShadow: `0 0 5px #fdbd48, 0 0 8px #fdbd48`,
```

Comment:
These colors need to be defined in our theme (if we don't already have an appropriate orange somewhere)